lib/ocpp/v16/charge_point_impl.cpp

```diff
@@ -993,7 +993,14 @@ void ChargePointImpl::message_callback(const std::string& message) {
                 if (enhanced_message.messageType == MessageType::BootNotificationResponse) {
                     this->handleBootNotificationResponse(json_message);
                 } else {
-                    this->handle_message(enhanced_message);
+                    if (enhanced_message.messageType == MessageType::GetConfiguration ||
+                        enhanced_message.messageType == MessageType::ChangeConfiguration ||
+                        enhanced_message.messageType == MessageType::TriggerMessage) {
+                        this->handle_message(enhanced_message);
+                    } else {
+                        auto call_error = CallError(enhanced_message.uniqueId, "InPendingState", "", json({}, true));
```

Comment:
I think this InPendingState error code does not exist, should probably be something like "GenericError" with a explanatory description